man/chpl.txt

```diff
@@ -65,8 +81,8 @@ OPTIONS
                     the Chapel compiler.
 
   --[no-]print-search-dirs   Print the module search path used to resolve module
-                    'use' statements. See the description of $CHPL_MODULE_PATH
-                    in the ENVIRONMENT section of this man page for more
+                    'use' statements. See the description of -M flag
+                    in the OPTIONS section of this man page for more
```

Comment:
Totally optional, but if you need to make changes anyway: "in the OPTIONS section" doesn't add much value given how long it is, so I'd probably just say "See the description of the -M flag for further details." (or "for more information.")